### ISSUE TITLE
GPU Training Time

### ISSUE THREAD

**Issue by jeonhuhuhu** [burchim/AVEC#9]
When you used 4 A100 GPUs, I wonder how many days it took to reach WER 37.5% [**Visual-only Efficient Conformer**] you only train visual information. 

Second question, is there a reason to use ResNet-18 for Visual Front-end when there are many good backbone models?

Or, is it to use author **Ma, Pingchuan's** model as it is to modify it comfortably?
https://github.com/mpc001/auto_avsr

**Comment by burchim** [comment]
Hi, the training time to reach 50 epochs using 4 A100 GPUs took around 1.5 days. The training time to complete each epoch was around 40 minutes.

For the second question, we use the same backbone as preceding works (the ResNet-18 similar to Ma Pingchuan's work) in order to compare with these methods more fairly. We did some experiments using larger ResNet models or using pretrained models but these did not bring performance improvements so we did not pursue the way of research.
